arangod/RocksDBEngine/RocksDBIndexCacheRefillFeature.cpp

```diff
@@ -165,6 +166,20 @@ or cache-enabled persistent indexes.)");
       .setIntroducedIn(31020)
       .setLongDescription(R"(The lower this number, the lower the impact of the
 index cache filling, but the longer it takes to complete.)");
+
+  options
+      ->addOption(
+          "--rocksdb.auto-refill-index-caches-on-followers",
+          "Whether or not to automatically (re-)fill the in-memory index "
+          "caches on followers as well.",
+          new options::BooleanParameter(&_autoRefillOnFollowers),
+          arangodb::options::makeFlags(options::Flags::DefaultNoComponents,
+                                       options::Flags::OnDBServer,
+                                       options::Flags::OnSingle))
+      .setIntroducedIn(31005)
+      .setLongDescription(R"(Set this to `false` to only (re-)fill in-memory
+index caches on leaders and save memory on followers. 
+Note that the value of this option should be identical for all DBServers.)");
```

Comment:
We really need cluster wide options. 
It's not first case when we want to force user set same value per all DBServers/Coordinators

Reply:
It is not critical if this option is set to different values on different DB servers, i.e. there will be no crashes, no data loss etc. if the option is set to different values.
It's just the functionality then won't work as desired.

I think I don't want to introduce (forced) cluster-wide options now as part of this PR.

Reply:
Of course, it's not message for some immediately action. Just thoughts